Resources/Prototypes/Entities/Structures/Storage/Closets/Lockers/base_structurelockers.yml

```diff
@@ -41,6 +41,16 @@
           SheetSteel1:
             min: 1
             max: 2
+  - type: Construction
+    graph: LockerGeneric
+    node: done
+    containers:
+    - entity_storage
+  - type: Reflect
```

Comment:
why is this in a construction pr

Reply:
it is copied from the crate code. Might as well add it